docker-compose.yml

```diff
@@ -65,6 +70,19 @@ services:
         networks:
             - diagnostic
 
+    mongo:
+        image: mongo
+        container_name: 'diagnostic-mongo'
+        restart: always
+        environment:
+            MONGO_INITDB_DATABASE: admin
+            MONGO_INITDB_ROOT_USERNAME: root
+            MONGO_INITDB_ROOT_PASSWORD: root
+        networks:
+            - diagnostic
+        volumes:
+            - ./util/mongo/mongo-init.js:/docker-entrypoint-initdb.d/mongo-init.js:ro
```

Comment:
Please create directory `.docker` in root and write here all files for docker (`mongo-init.js`).